ui/src/components/settings/sub_pages/licenses.rs

```diff
@@ -9,20 +9,23 @@ use crate::components::settings::SettingSection;
 #[allow(non_snake_case)]
 pub fn Licenses(cx: Scope) -> Element {
     log::trace!("Licenses settings page rendered.");
+    const MIT_URL: &str = "https://mit-license.org/";
```

Comment:
this goes to this

<img width="1199" alt="Captura de ecrã 2023-12-19, às 16 52 33" src="https://github.com/Satellite-im/Uplink/assets/29093946/748ac106-a635-4bcc-a4fe-b9c7985cb651">

we should use this url instead

https://github.com/Satellite-im/Uplink/blob/dev/LICENSE